### PR TITLE
Add api getCurrentFungibleAssetBalances

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Add api query - getCurrentFungibleAssetBalances

### Test Plan

pnpm test
pnpm fmt

### CODE REVIEW

**Comment by 0xjinn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #73** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/73" target="_blank">Graphite</a> 
    * **PR #78** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/78" target="_blank">Graphite</a> 
      * **PR #79** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/79" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/79?utm_source=stack-comment).

File: src/api/fungible_asset.ts [hunk 1]
```diff
@@ -36,4 +46,48 @@ export class FungibleAsset {
...
+   * It can be filtered by owner address, asset type, and type.
+   *
+   * @param options.pagination Where offset and limit can be specified
+   * @param options.whereCondition Where condition to filter the results
+   * @returns getFungibleAssetMetadata A list of fungible asset metadata
+   */
+  async getFungibleAssetActivities(args: {
+    options?: {
+      pagination?: PaginationArgs;
+      whereCondition?: {
```
Comment (0xmaayan) [inline src/api/fungible_asset.ts:63]: same comment as in the other PR https://github.com/aptos-labs/aptos-ts-sdk/pull/78/files#r1361610560

Comment (0xjinn) [reply src/api/fungible_asset.ts]: Yep removed and converted to using bool exp

File: tests/e2e/api/fungible_asset.test.ts [hunk 2]
```diff
@@ -38,4 +41,22 @@ describe("FungibleAsset", () => {
...
+    const APT_COIN_TYPE = "0x1::aptos_coin::AptosCoin";
+    const data = await aptos.getCurrentFungibleAssetBalances({
+      options: {
+        where: {
+          owner_address: { _eq: userAccount.accountAddress.toString() },
+          asset_type: { _eq: APTOS_COIN },
+        } as CurrentFungibleAssetBalancesBoolExp,
+      },
+    });
+    expect(data.length).toEqual(1);
```
Comment (heliuchuan) [inline tests/e2e/api/fungible_asset.test.ts:58]: todo, add test where length > 1

**Review by heliuchuan — APPROVED** [review]

File: src/api/aptos.ts [hunk 3]
```diff
@@ -50,6 +53,7 @@ export class Aptos {
     this.collection = new Collection(this.config);
     this.event = new Event(this.config);
     this.faucet = new Faucet(this.config);
+    this.fungible_asset = new FungibleAsset(this.config);
```
Comment (0xmaayan) [inline src/api/aptos.ts:56]:
```suggestion
    this.fungibleAsset = new FungibleAsset(this.config);
```

**Review by 0xmaayan — APPROVED** [review]

File: src/api/aptos.ts (hunk 3 above)
Comment (0xjinn) [reply src/api/aptos.ts:56]: Ahh I will update to use camelcases.

**Pull request merged**